### ISSUE TITLE
[Question]: Why output arrow instead of string like "left, right"

### ISSUE THREAD

**Issue by TianzeXie** [InternRobotics/InternNav#345]
### Question

I think the string output could leverage the base model's ability efficiently because the base model is already understand the meaning of these string and these string could align with the input instruction when such string exist in it, is it?

**Comment by Tai-Wang** [comment]
It is a design inherited from [StreamVLN](https://github.com/OpenRobotLab/StreamVLN) and is just used to save output tokens for faster inference. I agree with the idea that the string output can yield better alignment with the pretrained behavior. You can feel free to have a try to replace it during training, and I am also curious about what would happen :)

**Comment by TianzeXie** [comment]
> It is a design inherited from [StreamVLN](https://github.com/OpenRobotLab/StreamVLN) and is just used to save output tokens for faster inference. I agree with the idea that the string output can yield better alignment with the pretrained behavior. You can feel free to have a try to replace it during training, and I am also curious about what would happen :)

got it, thank you. I tried it and the loss converged much faster
